mission_control/navigator_launch/launch/perception.launch

```diff
@@ -4,4 +4,5 @@
   <!--node name="start_gate_vision" pkg="navigator_controller" type="start_gate_manual.py" respawn="True"/-->
 
   <node name="scan_the_code" pkg="navigator_vision" type="model_detector.py" respawn="True"/>
+  <node name="pinger_finder" pkg="navigator_vision" type="pinger_finder.py" />
```

Comment:
That's what it used to be, but that's too general, there's lidar perception, computer vision perception, and now sonar perception.